components/utilities/layout.js

```diff
@@ -110,7 +110,7 @@
                 layoutNodes.style.height = height;
 
                 // use the Facebook CSS goodness
-                computeLayout(layoutNodes);
```

Comment:
They moved to exposing an object with a few functions, rather than a single function. But they kept with the rather ambiguous name of computeLayout for the entry point into their API.

So, we could keep with their naming, resulting in:

    computeLayout.computeLayout(...)

I'm pretty happy either way!